packages/assistify-help-request/server/methods/createRequest.js

```diff
@@ -1,101 +1,201 @@
-Meteor.methods({
-	createRequest(name, expertise = '', openingQuestion, members = [], environment) {
-		const requestTitle = name;
-		check(name, String);
-		check(expertise, String);
-
-		const getNextId = function() {
-			// return HelpRequestApi.getNextAssistifyRoomCode();
-			const settingsRaw = RocketChat.models.Settings.model.rawCollection();
-			const findAndModify = Meteor.wrapAsync(settingsRaw.findAndModify, settingsRaw);
-
-			const query = {
-				_id: 'Assistify_Room_Count'
-			};
-
-			const update = {
-				$inc: {
-					value: 1
-				}
-			};
+import {RocketChat} from 'meteor/rocketchat:lib';
 
-			const findAndModifyResult = findAndModify(query, null, update);
 
-			return findAndModifyResult.value.value;
-		};
+class CreateRequestBase {
```

Comment:
Is this rearrangement necessary? Did you check if your changes are adapted to all files which use this method?

Reply:
@ThomasRoehl Yes, this is planned re-factoring.